CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 
     **Breaking** - removed build variants meta tasks. Minimum supported AGP version is 4.0.0.
   - Updated shadow plugin to 6.1.0 version.
+  - Updated Gradle to `6.7.1` version
+  - Set minimal supported Gradle version to `6.5.1`
```

Comment:
Seems like a pretty restricted version range to support. Didn't we want to support to back to 6.0?

Reply:
Updated commit to support it back to 6.0: cde95a1